### DISCUSSION TITLE
Prepared statements ballooning on 2.18?

### DISCUSSION THREAD

**Discussion by rasberrylasagna** [prisma/prisma#6363] [Q&A] [answered]
Hey all,

I'm running prisma on MySQL, and after updating to 2.18, and 2.20.1 I'm running into issues where my Prepared_stmt_count global MySQL variable is increased by a huge factor compared to 2.14, which leads to me hitting the MAX_PREPARED_STATEMENTS. It seems like the prepared statements aren't being flushed, or cleaned.

This is not a problem on 2.14, and was wondering if there was anything I could do to get the old behavior back.

Thank you!

**Comment by janpio** [comment, marked as answer]
Please open an issue and provide some more information what your application is doing. Optimally you could also output the queries generated from each version and compare them (see docs for how to log queries).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
